src/govuk/components/character-count/character-count.unit.test.mjs

```diff
@@ -36,22 +36,12 @@ describe('CharacterCount', () => {
 
     describe('i18n', () => {
       describe('JavaScript configuration', () => {
-        it('overrides the default translation keys', () => {
-          const component = new CharacterCount(document.createElement('div'), {
-            i18n: { charactersUnderLimit: { one: 'Custom text. Count: %{count}' } },
-            'i18n.charactersOverLimit.other': 'Different custom text. Count: %{count}'
-          })
-
-          expect(component.formatCountMessage(1, 'characters')).toEqual('Custom text. Count: 1')
-          expect(component.formatCountMessage(-10, 'characters')).toEqual('Different custom text. Count: 10')
-          // Other keys remain untouched
-          expect(component.formatCountMessage(10, 'characters')).toEqual('You have 10 characters remaining')
-        })
```

Comment:
**issue** Argh, I think there was a misunderstanding about what I was proposing to drop about that test. It was just the part about 'i18n.charactersOverLimit.other' that could be dropped as we no longer use them.

The test in itself remains relevant to check how options get merged with our defaults, I _think_. 

```javascript
        it('overrides the default translation keys', () => {
          const component = new CharacterCount(document.createElement('div'), {
            i18n: { charactersUnderLimit: { one: 'Custom text. Count: %{count}' } },
          })

          expect(component.formatCountMessage(1, 'characters')).toEqual('Custom text. Count: 1')
          
          // Other keys remain untouched
          expect(component.formatCountMessage(10, 'characters')).toEqual('You have 10 characters remaining')
        })
```

Reply:
Hey that's fine, don't worry, I'll stick it back in 😊

Reply:
Sorted 👍